7/17 (#213)
dolthub/weekly-updates: emails/2026-07-17.md

This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 

> I've been in Seattle all week and of course it rained. Never change Pacific Northwest. Never change.

Reviewer: ```suggestion
I've been in Seattle all week, and of course it rained. Never change, Pacific Northwest. Never change.
```

### Dolt in Four Flavors

Most of you know by now that Dolt comes in four flavors: [Dolt](https://github.com/dolthub/dolt), [Doltgres](https://github.com/dolthub/doltgresql), [DoltLite](https://github.com/dolthub/doltlite), and [Dumbo](https://github.com/dolthub/dumbodb). How do you know which one is right of you? [This article](https://www.dolthub.com/blog/2026-07-16-dolt-in-4-flavors/) tells you. 

### Dolt + Team City

> One of our long time power users built a Dolt Team City plugin. Have you ever wanted a Continuous Integration GUI that works with Dolt natively? With the Dolt Team City Plugin your wish has been answered. [Elian](https://www.dolthub.com/team#elian) [reviewed the plugin](https://www.dolthub.com/blog/2026-07-13-dolt-teamcity-vcs-plugin/) which also acts as a getting started guide. 

Reviewer: ```suggestion
One of our long time power users built a Dolt Team City plugin. Have you ever wanted a Continuous Integration GUI that works with Dolt natively? With the Dolt Team City Plugin your wish has been answered. [Elian](https://www.dolthub.com/team#elian)'s [review of the plugin](https://www.dolthub.com/blog/2026-07-13-dolt-teamcity-vcs-plugin/) also acts as a getting started guide. 
```

### Dolt in the News

> Dolt is showing up on the internet far more frequently these days. Thanks for writing or making videos about us! Every month, [Nick](https://www.dolthub.com#nick) picks a piece of content and writes a response. This month, [he wrote about a video produced by Better stack](https://www.dolthub.com/blog/2026-07-15-what-people-are-saying-better-stack/). Unsurprisingly, he wasn't happy with user comments on the internet.

Reviewer: ```suggestion
Dolt is showing up on the internet far more frequently these days. Thanks for writing or making videos about us! Every month, [Nick](https://www.dolthub.com#nick) picks a piece of content and writes a response. This month, [he wrote about a video produced by Better Stack](https://www.dolthub.com/blog/2026-07-15-what-people-are-saying-better-stack/). Unsurprisingly, he wasn't happy with user comments on the internet.
```
